### PR TITLE
[stable/traefik] add customizable entrypoint values for http/s/n/traefik

### PR DESCRIPTION
**Pull request by stevenolen**
#### What this PR does / why we need it:

Makes the listener address for http,https,httpn and traefik(dashboard) entrypoints customizable, setting the default to the existing hard-coded values to preserve backwards compatibility.

#### Special notes for your reviewer:

An open PR exists (#11164) to modify these ports away from privileged ports, but also with hard-coded values. This instead makes the values settable by the chart user. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [ ] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by stevenolen** [comment]
It just occurred to me that the address format is used in one place, and port-as-integer in another. a bit lame for me to miss that. I'm going to close this and re-submit when fixed.

**Comment by pho-enix** [comment]
Hi @stevenolen . Nice to see your PR. I like it more than my #11164 .

Just noticed that you were missing `templates/dashboard-service.yaml`

```
  ports:
  - name: dashboard-http
    port: 80
    targetPort: 8080
```

**Comment by stevenolen** [comment]
@pho-enix thanks for catching that one!

I added a template that translate the entrypoint address format needed for the conf file to a port that we can use in other places. Perhaps in hindsight asking the template user for a port and then generating the address string where needed is better, but this should work as well.

**Comment by stevenolen** [comment]
got a little cavalier and made the load balancer ports customizable as well, since they were hard-coded and we use things a little differently in our environment.

File: stable/traefik/values.yaml [hunk 1]
```diff
@@ -5,6 +5,14 @@ imageTag: 1.7.7
 serviceType: LoadBalancer
 # loadBalancerIP: ""
 # loadBalancerSourceRanges: []
+# default listener ports for load balancer
```
Comment (pho-enix) [inline stable/traefik/values.yaml:8]: I would prefer to have it under `service.ports.http`. Then all the service related pieces would be in one place:

```
service:
  ports:
    http: 80
    https: 443
  nodePorts:
    http: 30080
    https: 30443
```

**Comment by pho-enix** [comment]
Hi @stevenolen what's the status of this one? Will you fix the issues and can I close #11164 in favor of this?

**Comment by szubster** [comment]
@stevenolen Can we push it?

**Comment by stevenolen** [comment]
hi @szubster, finally getting back to this! I'll push a tweak today to take in @pho-enix suggestions and rebase on latest master

**Comment by szubster** [comment]
@stevenolen Any news on this?

**Pull request closed without merging**